src/vehicle.cpp

```diff
@@ -7585,7 +7585,10 @@ void vehicle::calc_mass_center( bool use_precalc ) const
             // Sometimes flag is wrongly set, don't crash!
             m_part += p != nullptr ? p->get_weight() : 0_gram;
         }
-
+        if( vp.has_feature( VPFLAG_BOARDABLE ) ) {
```

Comment:
Looks like there's no need to check the passenger_flag as `get_passenger` in the `if` above will check it anyway so the 2 `if`s can be merged

Reply:
OK, I merged them.